streams/src/test/java/org/apache/kafka/streams/processor/internals/DefaultStateUpdaterTest.java

```diff
@@ -956,10 +947,6 @@ private void verifyUpdatingTasks(final Task... tasks) throws Exception {
                 VERIFICATION_TIMEOUT,
                 "Did not get all updating task within the given timeout!"
             );
-            assertTrue(updatingTasks.stream()
-                .allMatch(task -> task.isActive() && task.state() == State.RESTORING
-                    ||
-                    !task.isActive() && task.state() == State.RUNNING));
```

Comment:
It doesn't make sense to verify the state of a mock task, because the mock will always return what we specified it should return independently of what the state updater does with the task. Totally missed that until now 🙂 !